shippable.yaml

```diff
@@ -6,23 +6,26 @@ python:
 
 branches:
   only:
-    - unreviewed
+    - unreviewed*
     - unreviewed2
-    - devel
-    - test_devel
+    - dev*
 
 build:
   ci:
+    - sudo add-apt-repository ppa:ansible/ansible-2.6 -y
+    - sudo apt-get update
+    - sudo rm -rf /usr/local/lib/python2.7/dist-packages/ansible
     - sudo apt-get install python3-pip shellcheck ansible
     - pip3 install -r dev-requirements.txt
+    - sudo pip2 install -r dev-requirements.txt
     - "./lint-all-the-files.sh"
     - "shipctl get_integration_resource_field aws_mdd accessKey"
-    - "shipctl get_resource_env aws_mdd DEMO"
-    - "echo success"
-    - "true"
     - "export AWS_ACCESS_KEY_ID=$(shipctl get_integration_resource_field aws_mdd accessKey)"
     - "export AWS_SECRET_ACCESS_KEY=$(shipctl get_integration_resource_field aws_mdd secretKey)"
+    - "export AWS_REGION=us-east-2"
```

Comment:
Should the version in here be hardcoded? if we need to run this in a different region, we will need to change this

Reply:
Yes;  right now this is a fact and it has to be somewhere.  Later it might need to be iterated, however the CI system is the top level thing.